### PR TITLE
Allow excluding some active tasks from tremor test

### PR DESCRIPTION
**Pull request by Erin-Mounts**

### CODE REVIEW

File: ResearchKit/Common/ORKOrderedTask.m [hunk 1]
```diff
@@ -1239,179 +1240,198 @@ + (ORKOrderedTask *)tremorTestTaskWithIdentifier:(NSString *)identifier
...
-        step.image = [UIImage imageNamed:@"tremortest4a" inBundle:[NSBundle bundleForClass:[self class]] compatibleWithTraitCollection:nil];
-        step.auxiliaryImage = [UIImage imageNamed:@"tremortest4b" inBundle:[NSBundle bundleForClass:[self class]] compatibleWithTraitCollection:nil];
-        step.shouldTintImages = YES;
-        
-        ORKStepArrayAddStep(steps, step);
-    }
-    
-    {
-        ORKCountdownStep *step = [[ORKCountdownStep alloc] initWithIdentifier:@"countdown2"];
+    if (! (activeTaskOptions & ORKTremorActiveTaskOptionExcludeHandInLap)) {
```
Comment (syoung-smallwisdom) [inline ResearchKit/Common/ORKOrderedTask.m:1243]: Nit: I prefer the explicit `if ((a & b) == 0) then do` or else a macro 

```
#define ORKFLAG_ON(flag, bitMask) ((flag & bitMask) == bitMask)
```

b/c I find it's a little easier for junior devs to grok.

**Comment by syoung-smallwisdom** [comment]
looks great, less filling

File: ResearchKit/Common/ORKOrderedTask.m (hunk 1 above)
Comment (Erin-Mounts) [inline ResearchKit/Common/ORKOrderedTask.m:1243]: (a) how are those junior devs ever going to learn proper c idioms then :P 
and more relevantly (b) follows the style already used in this file for bit masks

Comment (syoung-smallwisdom) [inline ResearchKit/Common/ORKOrderedTask.m:1243]: good point. ;)

**Pull request merged**